packages/xcb-util-errors/xcb-util-errors.pacscript

```diff
@@ -0,0 +1,19 @@
+name="xcb-util-errors"
+gives="lib${name}"
+pkgver="1.0.1"
+pkgdesc="XCB errors library"
+homepage='https://cgit.freedesktop.org/xcb/util-errors/'
+makedepends=("libtool" "xcb-proto" "xutils-dev" "autoconf" "autogen" "pkg-config" "libxcb1-dev" "python3")
+url="https://xorg.freedesktop.org/archive/individual/lib/xcb-util-errors-${pkgver}.tar.xz"
```

Comment:
`url="https://xorg.freedesktop.org/archive/individual/lib/${name}-${pkgver}.tar.xz"`

Reply:
It's not gonna change anytime soon.

Reply:
> It's not gonna change anytime soon.

its redundant code.

Reply:
> > It's not gonna change anytime soon.
> 
> its redundant code.

Your mom